lua/tardis/cl_hud.lua

```diff
@@ -0,0 +1,20 @@
+//TARDIS HUD
+
+function TARDISHUD()
```

Comment:
You can make this a local function within the hook.Add call below so we don't unnecessarily define this global function